src/goose/brokers/redis/batch.clj

```diff
@@ -0,0 +1,30 @@
+(ns goose.brokers.redis.batch
+  (:require
+    [goose.brokers.redis.commands :as redis-cmd]
+    [goose.defaults :as d]
+    [goose.job :as job]
+    [goose.retry :as retry]))
+
+(defn update-state [next]
+  (fn [{:keys [redis-conn] :as opts}
+       {:keys [id, batch-id] :as job}]
+    (if batch-id
+      (let [response (next opts job)
+            src (d/batch-set batch-id d/enqueued-job-set)
```

Comment:
Let's not presume the src & dst sets? Can be defined in line 29 by itself.

Reply:
dst set can change, so let's not define that already?

Reply:
+1 to both suggestions